### DISCUSSION TITLE
Lazer score screen rather uselessly shows global leaderboard top

### DISCUSSION THREAD

**Discussion by kiiyowastaken** [ppy/osu#28385] [Ideas] [answered]
Partially pasting from [forum post](https://osu.ppy.sh/community/forums/topics/1930474?n=1) since I was directed here.

Currently, upon finishing a map you're presented with your "score card", and to the left is an array of scores of the top of the global leaderboard. I feel like this is useless to the majority of the playerbase unless I am greatly underestimating the amount of people genuinely competing for top ~10 on beatmaps.

Is this because loading the scores of "surrounding" players, i.e displaying ~100 scores below and another ~100 above the player's score is too taxing processing or network/server wise (?), or for some other reason?

Would it be possible to display friend (maybe country?) scores if the player has osu!supporter?

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/26331
